### PR TITLE
New RT & BT env for opencti project & updated for newrelic-java-agent

### PR DESCRIPTION
**Pull request by NeerajK007**
New RT & BT env for opencti project.
update for newrelic-java-agent RT & BT env - added package manager & IDE

### CODE REVIEW

File: opencti/0.0.1/besman-opencti-BT-env-config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+BESMAN_ORG: Be-Secure
+BESMAN_OSSP: opencti
+BESMAN_OSSP_CLONE_PATH: $HOME/$BESMAN_OSSP
+BESMAN_ANSIBLE_ROLES_PATH: $BESMAN_DIR/tmp/$BESMAN_OSSP/roles
+BESMAN_ANSIBLE_ROLES: Be-Secureansible-role-oah-vscode
```
Comment (asa1997) [inline opencti/0.0.1/besman-opencti-BT-env-config.yaml:6]: Incorrect format.

File: newrelic/0.0.1/besman-newrelic-java-agent-BT-env.sh [hunk 2]
```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+function __besman_install_newrelic-java-agent-BT-env
+{
+    echo "hi, setting up the newrelic-java-agent-BT-env."
```
Comment (asa1997) [inline newrelic/0.0.1/besman-newrelic-java-agent-BT-env.sh:5]: Not required. 
echo statements should be __besman_echo statements

File: opencti/0.0.1/besman-opencti-BT-env.sh [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+    sleep 30 # Wait for the server to start (you may adjust the duration if needed)
+    firefox http://localhost:4000
+
+    echo "OpenCTI Frontend is now running!"
+    echo "4 ------- front-end setup completed...."
+    print "/n"
+}
+
+function __besman_uninstall_opencti-BT-env {
```
Comment (asa1997) [inline opencti/0.0.1/besman-opencti-BT-env.sh:222]: Are we not removing all the installed components here?

File: opencti/0.0.1/besman-opencti-BT-env.sh [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+        __besman_gh_clone "$BESMAN_ORG" "$BESMAN_OSSP" "$BESMAN_OSSP_CLONE_PATH"
+    fi
+
+    # Please add the rest of the code here for installation
+    # opencti-dev environment script
+    echo "1 ------- opencti dev modules setup..."
+    #!/bin/bash
+
+    # Function to install Docker
+    install_docker() {
```
Comment (asa1997) [inline opencti/0.0.1/besman-opencti-BT-env.sh:25]: Check if it can be done using the docker role.

**Comment by NeerajK007** [comment]
closing this and rasing a new pull request

**Pull request closed without merging**